test/bundlers.test.js

```diff
@@ -1,38 +1,35 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const fp = require('../plugin')
 
-test('webpack removes require.main.filename', (t) => {
+test('webpack removes require.main.filename', async (t) => {
```

Comment:
Why have all of the test function been refactored to async functions?

Reply:
@jsumners Thank you for your question. You are right to notice that all test functions have been converted to async functions. While it's not strictly necessary for the migration from `tap` to `node:test`, I chose to do so due to these reasons:

1. Consistency - It provides a uniform pattern across all tests.
2. Future-proofing - It allows for easier integration of async operations in the future if needed.
3. Minimal impact: The change doesn't negatively affect the tests' functionality or performance.

However, if the team prefers to keep the tests as close to the original implementation as possible, we can certainly revert this change and only use async functions where necessary.

Reply:
Unless there is a need for the functions to be async, please do not make them so.